package.json

```diff
@@ -39,13 +39,13 @@
     "js-minify": "npm-run-all --parallel js-minify-*",
     "js-minify-standalone": "terser --compress typeofs=false --mangle --comments \"/^!/\" --source-map \"content=dist/js/bootstrap.js.map,includeSources,url=bootstrap.min.js.map\" --output dist/js/bootstrap.min.js dist/js/bootstrap.js",
     "js-minify-bundle": "terser --compress typeofs=false --mangle --comments \"/^!/\" --source-map \"content=dist/js/bootstrap.bundle.js.map,includeSources,url=bootstrap.bundle.min.js.map\" --output dist/js/bootstrap.bundle.min.js dist/js/bootstrap.bundle.js",
-    "js-test": "npm-run-all js-test-karma* js-test-integration",
+    "js-test": "npm-run-all --parallel js-test-karma* --serial js-test-integration",
```

Comment:
We could probably run `js-test-integration` in parallel too, but I thought I'd leave this for another time.